packages/components/src/tooltip/index.tsx

```diff
@@ -112,7 +113,7 @@ function UnforwardedTooltip(
 			{ isOnlyChild && ( text || shortcut ) && (
 				<Ariakit.Tooltip
 					{ ...restProps }
-					className="components-tooltip"
+					className={ className === undefined ? "components-tooltip" : "components-tooltip " + className }
```

Comment:
This can be done with `clsx`:

```suggestion
					className={ clsx( 'components-tooltip', className ) }
```

You'll need to add an import at the top:

```
import clsx from 'clsx';
```